proto/c1/config/rules.proto

```
@@ -0,0 +1,194 @@
+    // supported).
+    bool   unique    = 3;
+    StringRules item_rules = 4;
+    // IgnoreEmpty specifies that the validation rules of this field should be
+    // evaluated only if the field is not empty
+    bool ignore_empty = 5;
+    bool is_required = 6;
+    // repeated string eq = 7;
+}
+
+enum WellKnownString {
+    WELL_KNOWN_STRING_UNKNOWN = 0;
```

Buf (1.47.2)

Enum zero value name "WELL_KNOWN_STRING_UNKNOWN" should be suffixed with "_UNSPECIFIED".

(ENUM_ZERO_VALUE_SUFFIX)